libs/dataset_deployer.py

```diff
@@ -97,6 +98,10 @@ def write_nested_csv(
             # value for a, (i.e. {a: None}), don't write a, as it's not in the header.
             flattened_row = {k: v for k, v in flattened_row.items() if k in header}
             flattened_row = {k: v for k, v in flattened_row.items() if not pd.isnull(v)}
+            flattened_row = {
+                k: v.value if isinstance(v, enum.Enum) else v for k, v in flattened_row.items()
```

Comment:
maybe replace `k: v.value if ...` with `k: v` if https://github.com/covid-projections/covid-data-public/blob/master/covidactnow/datapublic/common_fields.py#L20 on the enum class gets writerow to do what you want.